### DISCUSSION TITLE
resource mass/volume

### DISCUSSION THREAD

**Discussion by zer0Kerbal** [zer0Kerbal/TheGoldStandard#14] [Ideas]
## Currently the resource mass/volume is set like this
1. [goldOre]
   - mass (density): 0.0105
   - volume: 1 liter per unit (if I understand correctly)
   - unitCost: 50
2. [gold]
   - mass (density): 0.015
   - volume: 1 liter per unit (if I understand correctly)
   - unitCost: 500
3. [Unobtainium]
   - mass (density): 0.035
   - volume: 1 liter per unit (if I understand correctly)
   - unitCost: 50000

## comparisons:
1. [Ore] (stock)
   - mass (density): 0.010
   - volume: 5 liter per unit (if I understand correctly)
   - unitCost: 0.02
2. [Lead] (CRP)
   - mass (density): 0.01134
   - volume: 1 liter per unit (if I understand correctly)
   - unitCost: 0.067392
   
## changes proposed:
1. [goldOre]
   - mass (density): 0.015 from 0.0105 _(it contains gold and black sand)_
   - volume: 5L from 1L to match stock Ore
2. [gold]
   - mass (density): 0.0193 up from 0.015 (99.99% pure - rounded down the 0.00002)
      - Lead has a greater atomic mass than gold (196.97 VS 207.2) but yet, 
      - Gold is nearly twice as heavy as lead (19320 kg/cu.m VS 11340 kg/cu.m)
      - tank's `maxAmount` divided by a factor of 5 since tank is based upon `Ore` which has a volume of 5 (IIRC that is 5L per unit)
3. [Unobtainium]
      - tank's `maxAmount` divided by a factor of 5 since tank is based upon `Ore` which has a volume of 5 (IIRC that is 5L per unit)

**Comment by reseune** [comment]
I admit total ignorance here, but if the volume for gold ore is changing from 1 unit per liter to 5 per liter, that means that more gold ore will fit in the same space, correct? If this is true, then why would processing the gold ore into gold make it take up greater volume? My understanding is likely totally wrong though. 😂

**Reply by zer0Kerbal** [reply]
> I admit total ignorance here, but if the volume for gold ore is changing from 1 unit per liter to 5 per liter, that means that more gold ore will fit in the same space, correct? If this is true, then why would processing the gold ore into gold make it take up greater volume? My understanding is likely totally wrong though. 😂

@reseune I thought the same as well, however my current thoughts are that it would actually be 5L per unit of `goldOre` - so it would only affect how much each tank carries (and of course the total mass when full). wouldn't change the `goldOre` tanks capacity since `Ore` is also 5; rather it would take the `gold` and `Unobtainium` tanks and reduce their maxAmount's by a factor of 5 since each of those resources are a 1 instead of a 5 - which will happen anyways.

> p.s. I noticed the original post read `1 unit per liter`, and it should have been `1 liter per unit`. fixed now.

**Comment by reseune** [comment]
Copy. That makes sense, or at least, enough sense to me. 😂
________________________________
